src/mococrw/hsm.h

```diff
@@ -109,7 +109,7 @@ class HsmEngine : public HSM
     /** Token label used to uniquely identify a token on which objects reside */
     const std::string _tokenLabel;
     /** Pin to access PKCS11 Engine. */
-    const std::string _pin;
+    std::string _pin;
```

Comment:
Why not remove field in class? _ENGINE_ctrl_cmd_string(_engine.get(), "PIN", _pin); is called in constructor.